### DISCUSSION TITLE
Please adding directly links on post on Mod center

### DISCUSSION THREAD

**Discussion by thomasbnt** [forem/forem#19746] [Feature Requests]
Hi, I don't know why this was removed. 
To open the article from the mod center, we have to go through a not great passage, either through the tag or through the person's profile while being careful to open it in a new tab.

Previously, clicking on the post title would open it wide. Now you can't, and it's a bit sad and frustrating not to be able to.

![image](https://github.com/forem/forem/assets/14293805/bcf6b685-8f9b-4000-a225-0a7850c7da13)

**Comment by mikemike** [comment]
+1 on this, a "View Article" button to open the actual article in a new tab would be ideal.
